mud-contracts/world/src/modules/smart-gate/systems/SmartGateSystem.sol

```diff
@@ -228,7 +227,7 @@ contract SmartGateSystem is EveSystem, AccessModified {
 
     // Sum of squares (distance squared in meters)
     uint256 distanceSquaredMeters = (dx * dx) + (dy * dy) + (dz * dz);
-    return distanceSquaredMeters <= (maxDistance * maxDistance);
+    return distanceSquaredMeters <= maxDistance;
```

Comment:
```suggestion
    return distanceSquaredMeters <= (maxDistance * maxDistance);
```

Reply:
Another merge overwrite artifact, I will fix, thanks!

Reply:
fixed in commit 6d7f21bfb9ff1ed3565a4c68b7bb29811cc2a75e